lib/chat_web/channels/room_channel.ex

```diff
@@ -2,13 +2,13 @@ defmodule ChatWeb.RoomChannel do
   use ChatWeb, :channel
 
   @impl true
-  def join("room:lobby", _payload, socket) do
-    #if authorized?(payload) do
+  def join("room:lobby", payload, socket) do
+    if authorized?(payload) do
```

Comment:
`authorized?` always returns `true` (see code below).

Reply:
this `authorized?/1` is a placeholder. 👌